components/form-builder/app/DownloadTable.tsx

```diff
@@ -0,0 +1,259 @@
+import React from "react";
+import { VaultSubmissionList } from "@lib/types";
+import { useTranslation } from "react-i18next";
+import { ExclamationText } from "./shared";
+import { SkipLinkReusable } from "@components/globals/SkipLinkReusable";
+
+interface DownloadTableProps {
+  submissions: VaultSubmissionList[];
+  checkedItems: Map<string, boolean>;
+  setCheckedItems: React.Dispatch<React.SetStateAction<Map<string, boolean>>>;
+  getCheckedItems: () => Map<string, boolean>;
+  handleDownload: React.MouseEventHandler<HTMLButtonElement>;
+}
+
+export const DownloadTable = ({
+  submissions,
+  checkedItems,
+  setCheckedItems,
+  getCheckedItems,
+  handleDownload,
+}: DownloadTableProps) => {
+  const { t } = useTranslation("form-builder");
+  const DOWNLOAD_OVERDUE = 15;
+  const CONFIRM_OVERDUE = 15;
+
+  function getDaysPassed(date: Date): number {
+    const dateCreated = new Date(date);
+    const dateToday = new Date();
+    const dateDiff = Math.abs(Number(dateToday) - Number(dateCreated));
+    const daysPassed = Math.ceil(dateDiff / (1000 * 60 * 60 * 24));
+    return daysPassed;
+  }
+
+  // Format date for: DD/MM/YYYY
+  function formatDate(date: Date): string {
+    const dateObj = new Date(date);
+    const day = String(dateObj.getDate()).length <= 2 ? `0${dateObj.getDate()}` : dateObj.getDate();
+    const month =
+      String(dateObj.getMonth()).length <= 2
+        ? `0${dateObj.getMonth() + 1}`
+        : dateObj.getMonth() + 1;
+    const year = dateObj.getFullYear();
+    return `${day}/${month}/${year}`;
+  }
+
+  function formatStatus(vaultStatus: string) {
+    switch (vaultStatus) {
+      case "New":
+        return (
+          <span className="p-2 bg-[#ecf3eb] text-[#0c6722]">
+            {t("downloadResponsesTable.status.new")}
+          </span>
+        );
+      case "Downloaded":
+        return (
+          <span className="p-2 bg-[#dcd6fe]">{t("downloadResponsesTable.status.downloaded")}</span>
+        );
+      case "Confirmed":
+        return (
+          <span className="p-2 bg-[#e2e8ef]">{t("downloadResponsesTable.status.confirmed")}</span>
+        );
+      case "Problem":
+        return (
+          <span className="p-2 bg-[#f3e9e8] text-[#bc3332]">
+            {t("downloadResponsesTable.status.problem")}
+          </span>
+        );
+      default:
+        return (
+          <span className="p-2 bg-[#f3e9e8] text-[#bc3332]">
+            {t("downloadResponsesTable.unknown")}
+          </span>
+        );
+    }
+  }
+
+  function formatDownloadResponse({
+    vaultStatus,
+    createdAt,
+    downloadedAt,
+  }: {
+    vaultStatus: string;
+    createdAt?: Date;
+    downloadedAt?: Date;
+  }) {
+    if (vaultStatus === "New" && createdAt) {
+      const daysPassed = getDaysPassed(createdAt);
```

Comment:
Maybe for a future iteration these could be broken into components that can handle there own concerns (props) 

Would make things easier to test and read :)